src/message.ts

```diff
@@ -604,6 +604,34 @@ export class Message implements Sayable {
     ])
   }
 
+  /**
+   * Return the recalled message id
+   *
+   * Valid only if {@link Message#type} is equal to `MsgType.RECALLED`
+   *
+   * @returns {string} msgId
+   * @memberof Message
+   * @example
+   * ```javascript
+   * Bot.on('message', async m => {
+   *   if (m.type() === MsgType.RECALLED) {
+   *     await m.say(`You recalled a message, message ID：${m.getRevokeId()}`)
+   *   }
+   * })
+   * ```
+   */
+  public getRevokeId(): string {
```

Comment:
Why we need to add this new method?

I did not think it's necessary. 

Please correct me if I'm wrong.